src/ViewRequest.js

```diff
@@ -408,6 +408,9 @@ class ViewRequest extends React.Component {
 
     const tags = ((request && request.tags) || {}).tagList || [];
 
+    const requestStatus = get(request, ['status'], '-');
+    const isRequestClosed = requestStatus.startsWith('Closed');
```

Comment:
Hello, @Terala-Priyanka thank you. Could you please investigate possible of use closed request status from https://github.com/folio-org/ui-requests/blob/master/src/constants.js#L32 for comparison?
CC: @zburke , @artem-blazhko

Reply:
Sure, converting that to a function/list of statuses is a good idea to make sure open and closed are handled consistently across the application. I've advocated for a while (like, years) that the `status` attribute should itself have a boolean `isOpen` attribute to make these kinds of filters easier. Alas.

Reply:
@Dmitriy-Litvinenko  
Thank you for the observation. I have updated the code.